bot/utils/converters.py

```diff
@@ -1,11 +1,15 @@
+from datetime import datetime
+from typing import Tuple, Union
+
 import discord
-from discord.ext.commands.converter import MessageConverter
+from discord.ext import commands
 
 
-class WrappedMessageConverter(MessageConverter):
+class WrappedMessageConverter(commands.MessageConverter):
     """A converter that handles embed-suppressed links like <http://example.com>."""
 
-    async def convert(self, ctx: discord.ext.commands.Context, argument: str) -> discord.Message:
+    @staticmethod
```

Comment:
The staticmethod is making the following traceback appear
```
sir-lancebot    | Traceback (most recent call last):
sir-lancebot    |   File "/usr/local/lib/python3.8/site-packages/discord/ext/commands/core.py", line 451, in _actual_conversion
sir-lancebot    |     ret = await instance.convert(ctx, argument)
sir-lancebot    |   File "/bot/bot/utils/converters.py", line 20, in convert
sir-lancebot    |     return await super().convert(ctx, argument)
sir-lancebot    | TypeError: super(type, obj): obj must be an instance or subtype of type
```

Reply:
This was fixed in https://github.com/python-discord/sir-lancebot/pull/718/commits/8dd9e9cebd2496dc2e7e82569deb2fa9f1a5f89f.